service/cloudconfig/cloud_config.go

```diff
@@ -0,0 +1,33 @@
+package cloudconfig
+
+import (
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig"
+)
+
+type MasterExtension struct{}
+
+func (me *MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
+	return nil, nil
+}
+
+func (me *MasterExtension) Units() ([]k8scloudconfig.UnitAsset, error) {
+	return nil, nil
+}
+
+func (me *MasterExtension) VerbatimSections() []k8scloudconfig.VerbatimSection {
```

Comment:
We may need verbatim sections for the cloud provider support. Not certain yet so will also come later.